AsmHighlighter/AsmHighlighterMPLexer.lex

```diff
@@ -35,13 +35,14 @@
 
 
 binary              [0-1]
+octal               [0-7]
 digit               [0-9]
 alpha               [a-zA-Z_]
 exponent            [Ee]("+"|"-")?{digit}+
 floatsuffix         [fFhH]
 white_space         [ \t\v\n\f\r]
-hexdigit            [0-9a-fA-F]
-ABStar       [^\*\n]*
+hexdigit            [0-9a-fA-F]
+ABStar       [^\*\n]*

```

Comment:
looks like we have some line ending issues here
I normalized the line endings: https://github.com/Trass3r/AsmHighlighter/commit/8fc1b2a5e5dcb8b8eadbad5b2a68600a0da1ceb8
Please try a rebase.